src/Umbraco.Web/PublishedCache/NuCache/DataSource/AppSettingsNucachePropertyMapFactory.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Umbraco.Web.PublishedCache.NuCache.DataSource
+{
+    public class AppSettingsNucachePropertyMapFactory : INucachePropertyOptionsFactory
```

Comment:
This will be ok for testing for now until we progress this further but ultimately a lot of how property values are serialized/deserialized will need to be up to the property editor itself. We can of course have other config bits and pieces

Reply:
Fixed spelling. Not currently sure how to get the UI / property editor to configure, I took this approach as I wasn't sure how this feature would be supported by non-core property editors.

Reply:
Yep no worries gotta start somewhere. I have a couple hours tonight so gonna merge this in and then can go from there.